src/com/facebook/buck/rules/KnownBuildRuleTypes.java

```diff
@@ -513,9 +513,9 @@ static Builder createBuilder(
 
     SwiftLibraryDescription swiftLibraryDescription =
         new SwiftLibraryDescription(
+            cxxBuckConfig,
             cxxPlatforms,
-            platformFlavorsToAppleCxxPlatforms,
-            defaultCxxPlatform);
+            platformFlavorsToAppleCxxPlatforms, defaultCxxPlatform);
```

Comment:
Didn't notice that, IntelliJ should be smarter with style auto-formatting after refactoring.